src/support/variables/typography.scss

```diff
@@ -1,24 +1,30 @@
 // Typography variables
 
+@function font-size-to-rem($px, $baseFontSize) {
+  @return ($px / $baseFontSize) * 1rem;
+}
```

Comment:
We have a `rem(` mixin already for this in support.

Reply:
It sounds like we can either use the same, or if they're in different repos, maybe clone it and maintain both at the same time, since we don't have a share library of utilities, right?

Reply:
One thing I'd like to see us do in the implementation is, to not use the sass mixin at all. Currently it's using some [deprecated math sass functions](https://github.com/primer/css/pull/1452) that is blocking us from moving the build to dart-sass. If we could compute these new rem values and enter them directly that would be ideal. We could show our math in a CSS comment above each line to show that it's the `px` equivalent.

Reply:
Here a PR where the values are already computed to `rem`s: https://github.com/primer/css/pull/1538